Core/Base/ConsumerModule.py

```diff
@@ -0,0 +1,43 @@
+"""
+  DIRAC class which is the abstract class for the consumers
+
+  All DIRAC consumers must inherit from the basic class ConsumerModule
+  and override its methods
+"""
+
+from DIRAC import S_ERROR
+#from DIRAC.Core.Utilities.MQConnector import MQConnector
+
+class ConsumerModule(object):
+  """ Base class for all consumer modules
+
+      This class is used by the ConsumertReactor class to steer the execution of
+      DIRAC consumers.
+  """
+
+  def __init__( self ):
```

Comment:
I added CS parameter loading, but didnt strictly to AgentModule format - since I dont like it so much.